packages/flutter/lib/src/widgets/draggable_scrollable_sheet.dart

```diff
@@ -34,6 +34,56 @@ typedef ScrollableWidgetBuilder = Widget Function(
   ScrollController scrollController,
 );
 
+class DraggableScrollableSheetController {
```

Comment:
It's conventional to set initial values (eg `DraggableScrollableSheet.initialChildSize`) from a controller instead of from the widget consuming the controller (see `PageController` and `ScrollController`). Maybe it'd be good to do that here? But then should `minChildSize`, `snapSizes`, etc also be included here? Probably not-these would all be breaking changes and force developers to specify a controller even if all they want to do is set `min`, `max`, etc.

No call to action here-this is all just stuff to keep in mind.